src/consts/walletList/hosted.ts

```diff
@@ -55,9 +56,13 @@ export const FIGURE_HOSTED_TEST = {
       if (event) searchParams.append('event', event);
       if (redirectUrl) searchParams.append('redirectUrl', redirectUrl);
       const searchParamsString = searchParams.toString();
-      const url = `${FIGURE_HOSTED_WALLET_URL_TEST}${
-        searchParamsString ? `&${searchParamsString}` : ''
-      }`;
+      const overrideUrl = localStorage.getItem(
+        'FIGURE_HOSTED_WALLET_URL_TEST_OVERRIDE'
+      );
+      const baseUrl = overrideUrl
+        ? `${overrideUrl}/${FIGURE_HOSTED_WALLET_URL}`
```

Comment:
I feel like this localStorage value should be the whole url to allow full customization vs using the partial url from consts.